core/src/main/scala/za/co/absa/spline/harvester/logging/ObjectStructureDumper.scala

```diff
@@ -96,6 +96,7 @@ object ObjectStructureDumper {
                 try {
                   extractFieldValue(value, f.getName)
                 } catch {
+                  case e: LinkageError => s"! error occurred: ${e.toShortString}"
                   case NonFatal(e) => s"! error occurred: ${e.toShortString}"
```

Comment:
Just for the sake of brevity
```suggestion
                  case e@(_: LinkageError | NonFatal(_)) => s"! error occurred: ${e.toShortString}"
```